### ISSUE TITLE
Timer don't appears when constructing with commander

### ISSUE THREAD

**Issue by Anpix** [tgyhlsb/OgameTimer#1]
![image](https://cloud.githubusercontent.com/assets/6361359/21351675/fa435186-c6a4-11e6-90e2-1daabf39b742.png)

**Comment by JacobSmith16** [comment]
![ogame timer exp](https://cloud.githubusercontent.com/assets/24728868/21448586/8f4c9f22-c938-11e6-873f-c609fba858b8.PNG)
If you checked the feature and it doesn't work and breaks everything, don't blame me :)

Go to your extensions page (chrome://extensions) if on chrome and click on options then check enable "Construction queue (experimental, use at your own risk)".

**Comment by Anpix** [comment]
Yeah... I know that is experimental. So I came here on GitHub to say that, not in the comments of the app in Chrome Store.
I'm developer too, and I like to report every error (or improvement) found. This is the function of "issues" tab on GitHub.
I hope to help this project soon.
